src/test/scala/com/beachape/filemanagement/FileSystemWatchMessageForwarderActorSpec.scala

```diff
@@ -0,0 +1,82 @@
+package com.beachape.filemanagement
+
+import java.io.{File, PrintWriter}
+import java.nio.file.StandardWatchEventKinds._
+
+import akka.actor.ActorSystem
+import akka.testkit.{ImplicitSender, TestActorRef, TestKit}
+import com.beachape.FileSystemWatchMessageForwardingActor
+import com.beachape.filemanagement.Messages.EventAtPath
+import org.scalatest._
+
+import scala.concurrent.duration._
+
+/**
+ * Created by arunavs on 9/27/16.
+ *
+ * Test casses for FileSystemWatchActor
+ */
+
+class FileSystemWatchMessageForwarderActorSpec extends TestKit(ActorSystem("FSWatchTest")) with ImplicitSender
+    with WordSpecLike with Matchers with BeforeAndAfterAll with BeforeAndAfterEachTestData {
+
+  val locationDir = "./src/test/permdata"
+  val locationDirFileHandle = new File(locationDir)
+  locationDirFileHandle.deleteOnExit()
+  val allLocs = List("./src/test/permdata")
+
+  /**
+   * Note the passing of the `self` reference, where self
+   * is the actor for the test suite itself so we can use expect msg
+   */
+  val customActorRef = TestActorRef(new FileSystemWatchMessageForwardingActor(allLocs, self))
+
+  override def afterAll : Unit = {
+    TestKit.shutdownActorSystem(system)
+  }
+
+  "An FSWatchImproved actor" must {
+    "send a changed Dir Message of modified type file when an existing file is modified" in {
+      val existingFileName = "existingFile.txt"
+      val existingFile = new File(locationDir + File.separator + existingFileName)
+      printTestToFile(existingFile)
+      within(100 seconds) {
```

Comment:
The build is failing to compile because you're missing a required import. See [these lines](https://travis-ci.org/lloydmeta/schwatcher/jobs/177376741#L286-L300).